### PR TITLE
Feature/issues/505

### PR DESCRIPTION
**Pull request by Mizzick**
#505

### CODE REVIEW

**Comment by ameshkov** [comment]
How does it solve this issue?
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/505#issuecomment-295268212

File: Extension/lib/pages/popup.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ $(document).ready(function () {
             //add some delay for show popup size properly
             setTimeout(function () {
                 controller.resizePopupWindow();
-            }, 10);
+            }, 150);
```
Comment (ameshkov) [inline Extension/lib/pages/popup.js]: Huh, that's quite weird. Why 150?

Comment (Mizzick) [inline Extension/lib/pages/popup.js]: This solves the layout issue.

**Comment by Mizzick** [comment]
Cherry-picked to master without icons changing.

**Pull request closed without merging**